admin/client/src/components/ListGroup/LIstGroup.scss

```diff
@@ -0,0 +1,96 @@
+// List-group, based on Bootstraps list-group
+// TODO split out campaign specific css from generic list-group
+
+.list-group {
```

Comment:
This file has the first two letters capitalised by mistake. you can use `git mv LIstGroup.scss ListGroup.scss` to fix.